vllm_spyre/platform.py

```diff
@@ -209,28 +210,14 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         os.environ["VLLM_DT_MAX_BATCH_SIZE"] = str(
             max(vllm_config.scheduler_config.max_num_seqs, 2))
 
+        # Hardcode some things for granite-3.3-8b-instruct
+        cls.configure_granite_33_8b(vllm_config)
```

Comment:
Very much a style NIT and totally optional, but I think it reads better if the `if not cls.is_granite_33_8b(vllm_config.model_config):` check is moved here instead of embedded in `configure_granite_33_8b`.